app/src/main/java/com/android/streetworkapp/ui/event/EventOverview.kt

```diff
@@ -161,15 +168,23 @@ fun EventMap(park: Park) {
             Icons.Outlined.LocationOn,
             contentDescription = "location",
             modifier = Modifier.padding(horizontal = 8.dp).testTag("locationIcon"))
-        Text("at ${park.name}", modifier = Modifier.testTag("location"))
+        Text("at ${park?.name}", modifier = Modifier.testTag("location"))
       }
 
   GoogleMap(
       modifier = Modifier.testTag("googleMap").fillMaxSize(),
       cameraPositionState = cameraPositionState) {
         Marker(
-            contentDescription = "Marker",
-            state = MarkerState(position = LatLng(parkLatLng.latitude, parkLatLng.longitude)))
+            contentDescription = "Event location",
+            state = MarkerState(position = parkLatLng ?: LatLng(0.0, 0.0)),
```

Comment:
Suggestion: Since now the markers are blue on the main map, they could also be changed to blue on the event overview in order to make the UI coherent.

Reply:
You a right, i'll change it